synapse/tests/test_cortex.py

```diff
@@ -1708,3 +1709,31 @@ def ondel(node, tag, valu):
             nodes = await alist(core.eval('##baz.faz'))
             self.len(1, nodes)
             self.eq(nodes[0].ndef[1], 'visi')
+
+    async def test_cortex_latency(self):
+        ''' Issue a large snap request, and make sure we can still do stuff in a reasonable amount of time'''
+
+        async with self.getTestCore() as core:
+
+            async with await core.snap() as snap:
+
+                event = asyncio.Event()
+
+                async def add_stuff():
+                    event.set()
+                    ips = ((('inet:ipv4', x), {}) for x in range(20000))
+
+                    msgs = await alist(snap.addNodes(ips))
+                    print('got msgs', len(msgs))
```

Comment:
left over debugging? or is there an assertion we should have here?
or is this coro even expected to finish by the time we are tearing down the test cortex?

Reply:
Left over debug statement. Fixed. This is only expected to finish in the failing case.